mappings/net/minecraft/util/registry/DataPackLoadedRegistries.mapping

```diff
@@ -0,0 +1,37 @@
+CLASS net/minecraft/class_5458 net/minecraft/util/registry/DataPackLoadedRegistries
```

Comment:
This class name is a little verbose, maybe `DataPackRegistries`, or maybe `DynamicRegistries` with a javadoc comment saying what it is?

Reply:
Yes, `DynamicRegistries` will change